### PR TITLE
Awesome Books ES6 Modules

### PR DESCRIPTION
**Pull request by dselasea**
In this pull request I
- Created a module folder
- Broke my JavaScript code into ES6 modules
- Used Luxon package
- Used let and const the right way
- Used arrow functions

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+});
+
+contactlink.addEventListener('click', () => {
+  contact.style.display = 'block';
+  list.style.display = 'none';
+  add.style.display = 'none';
+  listlink.style.color = 'black';
+  contactlink.style.color = 'blue';
+  addlink.style.color = 'black';
+});
```
Comment (prolajumokeoni) [inline index.js]: - [ ] Please make sure your code is divided into reasonable modules, please separate different functionalities in your index.js into modules for example the navigation, add & remove book and display books can be separate modules

File: index.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    return;
+  }
+  savebook.BookData.forEach((item) => DisplayBooks(item));
+});
+
+setInterval(() => {
+  const dateTimeNow = DateTime.now();
+  document.querySelector('.date').innerHTML = dateTimeNow.toLocaleString(DateTime.DATETIME_MED);
+}, 1000);
+
```
Comment (prolajumokeoni) [inline index.js]: - [ ] Based on the requirement, please make sure the current date is displayed below the navigation bar,  currently your date is displayed on the footer please change it 👍🏿

File: index.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
```
Comment (prolajumokeoni) [inline index.js]: - [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @dselasea 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Navigation works properly
✔️  Users can add or remove a book



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @dselasea , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**